### ISSUE TITLE
Blogs UI

### ISSUE THREAD

**Issue by rohanshrimal** [KoreroInc/major-project#67]
### Issue Description

- Home Page of blogs should have all the related statistics associated with the blogs.

- In order to improve the accessibility of blogs, they should be shown in the form of tiles.
